Week2/homework2/queryfunctions.js

```diff
@@ -0,0 +1,84 @@
+const inquirer = require('inquirer');
+const { execQuery, con } = require('./connection');
+const {
+  Query1,
+  input1,
+  Query2,
+  input2,
+  Query3,
+  input3,
+  Query4,
+  input4,
+  Query5,
+  input5,
+} = require('./queries');
+
+const queryCapital = async () => {
+  const { countryName } = await inquirer.prompt(input1);
+  const result = await execQuery(Query1, countryName);
+  console.log(`The capital city of ${countryName} is ${result[0].capital}`);
+};
+const queryLanguages = async () => {
+  const { regionName } = await inquirer.prompt(input2);
+  const result = await execQuery(Query2, regionName);
+  console.log(
+    `Languages spoken in the region of ${regionName}:\n`,
+    result.map(item => Object.values(item).toString()),
+  );
+};
+const queryCities = async () => {
+  const { languageName } = await inquirer.prompt(input3);
+  const result = await execQuery(Query3, languageName);
+  console.log(`The language of ${languageName} is spoken in ${result[0].Number_Of_Cities} cities`);
+};
+const queryOfficiallanguages = async () => {
+  const { regionName, languageName } = await inquirer.prompt(input4);
+  const result = await execQuery(Query4, [regionName, languageName]);
+  const answer = result.length === 0 ? 'FALSE' : result.map(item => Object.values(item).toString());
+  console.log(`Countries  in the ${regionName}(where ${languageName} is official language):`);
+  console.log(answer);
+};
+const queryContinents = async () => {
+  const result = await execQuery(Query5);
+  Object.entries(result[0]).forEach(([key, value]) => console.log(`${key}: ${value}`));
+};
+const explanation = `           OPTIONS
+1. What is the capital of country X ? 
+2. List all the languages spoken in the region Y
+3. Find the number of cities in which language Z is spoken 
+4. Accept the region and language from the user.Are there any countries in 
+   this region with the given language as the official language ?  
+5. List all the continents with the number of languages spoken in each continent\n`;
+async function seedDatabase() {
+  try {
+    con.connect();
+
+    console.log(explanation);
+    const { choice } = await inquirer.prompt(input5);
```

Comment:
I love the arrow choice :) The program doesn't end when I select, anyway to make it end or give me back the menu ?

Reply:
Actually, I wanted to do it in a while loop. I wanted to execute program(asking queries/menu) until user select quit. But I could not manage it. 
Yes, you are right. After making a selection and allocating proper inputs, the program doe not quit. How can I solve it?

Reply:
Why are you executing your queryCapital func into the executeQuery statement ? executeQuery that you are using correctly after is supposed to be used with the sql statement, not a func. Try to remove the executeQuery and just await your queryCapital func .

Reply:
yes you are right. thanks